Source/EasyNetQ/ConnectionConfiguration.cs

```diff
@@ -35,6 +35,7 @@ public class ConnectionConfiguration
         public string Platform { get; set; }
         public string Name { get; set; }
         public bool UseBackgroundThreads { get; set; }
+        public bool UseWaitJoin { get; set; }
```

Comment:
Let's drop thread joining from this PR. It's a separate discussion about whether we need this or now.

Reply:
Ok

Reply:
Thanks.